ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```
@@ -886,6 +886,9 @@ tests:
   optional: true
   steps:
     cluster_profile: oci-assisted
+    env:
+      ASSISTED_CONFIG: |
+        NUM_WORKERS=3
     workflow: assisted-oci
```

@adriengentil looking the latest run[1], maybe we need a higher timeout for this job or workflow:
```suggestion
    workflow: assisted-oci
    timeout: 6h0m0s
```

[1] https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48274/rehearse-48274-pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted/1755533740458643456#1:build-log.txt%3A64-65